### PR TITLE
fix some deprecations and add upgrade instructions

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

### CODE REVIEW

**Review by Tobion — APPROVED** [review]

**Comment by xabbuh** [comment]
updated to take into account #31975/#32452 too

File: src/Symfony/Component/Lock/Store/MemcachedStore.php [hunk 1]
```diff
@@ -74,7 +74,7 @@ public function save(Key $key)
      */
     public function waitAndSave(Key $key)
     {
-        @trigger_error(sprintf('%s has been deprecated since Symfony 4.4 and will be removed in Symfony 5.0.', __METHOD__));
+        @trigger_error(sprintf('%s() is deprecated since Symfony 4.4 and will be removed in Symfony 5.0.', __METHOD__));
```
Comment (Simperfit) [inline src/Symfony/Component/Lock/Store/MemcachedStore.php]: I have updated all of this in a PR already ;)

**Review by Simperfit — APPROVED** [review]

File: src/Symfony/Component/Lock/Store/MemcachedStore.php (hunk 1 above)
Comment (xabbuh) [reply src/Symfony/Component/Lock/Store/MemcachedStore.php]: I must have missed that one. :) Do I am gonna rebase here afterwards.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**